client/me/security-2fa-enable/index.jsx

```diff
@@ -178,22 +178,54 @@ module.exports = React.createClass( {
 		}
 	},
 
+	getToggleLink: function() {
+		return (
+			<a
+				className="security-2fa-enable__toggle"
+				onClick={ function( event ) {
+					analytics.ga.recordEvent( 'Me', 'Clicked On Barcode Toggle Link', 'current-method', this.state.method );
+					this.toggleMethod( event );
```

Comment:
Minor thing, but I would put toggleMethod before the analytics.ga. Since it is more important then the analytics event.